scripts/update-fake-master.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kinit -k -t $KEYTAB $PRINCIPLE
+
+# Use the directory relative to this Jenkins job.
+BUILDPATH="${WORKSPACE}/go"
+mkdir -p $BUILDPATH
+cd $BUILDPATH
+export GOPATH="$( pwd )"
+WORKPATH="${BUILDPATH}/src/github.com/openshift/"
+mkdir -p $WORKPATH
+echo "GOPATH: ${GOPATH}"
+echo "BUILDPATH: ${BUILDPATH}"
+echo "WORKPATH ${WORKPATH}"
+
+# Ensure ssh-agent is running
+eval "$(ssh-agent -s)"
+
+# Add back deploy key for cloning/pushing openshift/ose
+ssh-add -D
+ssh-add ${HOME}/.ssh/id_rsa
+
+cd ${WORKPATH}
+rm -rf ose
+git clone git@github.com:openshift/ose.git
+pushd ose
+git remote add upstream git@github.com:openshift/origin
+git fetch --all
+
+# Check if there is anything to update.
+LATEST_TAG=$(git describe --abbrev=0 --tags)
+LATEST_TITO_COMMIT_MERGE=$(git log -1 $LATEST_TAG --pretty=%s)
+git checkout fake-master
+LATEST_TITO_COMMIT_REBASE=$(git log -1 HEAD --pretty=%s)
```

Comment:
No need to checkout:

    LATEST_TITO_COMMIT_MERGE=$(git log -1 origin/fake-master --pretty=%s)